projects/packages/videopress/src/client/block-editor/blocks/video/components/details-panel/index.tsx

```diff
@@ -72,6 +75,14 @@ export default function DetailsPanel( {
 				help={ descriptionHelp }
 			/>
 
+			{ !! chapter && ! isAutoGeneratedChapter && (
+				<Notice status="success" className="learn-how-notice" isDismissible={ false }>
+					<p className="learn-how-notice__message">
+						{ __( 'You already have a vtt file', 'jetpack-videopress-pkg' ) }
```

Comment:
Should we be more informative here, like "You already have chapter information on a attached VTT file. Adding chapters to the description will not change the original chapters", so the user knows that the chapters won't work from there?

Reply:
we've updated the description [after talking with VP folks](p1669201017490559-slack-C02LT75D3).

<img width="333" alt="image" src="https://user-images.githubusercontent.com/77539/203548350-873b29c9-7d28-4567-9729-00299cfb490a.png">

Addressed [here](https://github.com/Automattic/jetpack/pull/27544/commits/eeab29ee7bd8a88c80092cb392517b68bda8da30)